reporter/parca_reporter.go

```diff
@@ -69,6 +72,31 @@ type stack struct {
 	frameTypes []libpf.FrameType
 }
 
+type labelsKey struct {
+	pid util.PID
+	tid util.PID
+	comm string
```

Comment:
does comm really need to be a part of this key, isn't pid+tid already uniquely identifying the thread?

Reply:
yes, you're right.

However, we should still not cache comm, since it can change (e.g. if `pthread_setname_np` is called). But it doesn't need to be part of the key.